services/streammanager/kafka/kafkamanager.go

```diff
@@ -659,3 +660,15 @@ func getStatusCodeFromError(err error) int {
 	}
 	return 400
 }
+
+func getSSHConfig(c *config.Config) *client.SSHConfig {
```

Comment:
Would it make sense to be able to enable it globally, but also allow for destinationID-specific overrides? e.g.

```yaml
Router:
  KAFKA:
    ssh:
      enabled: false
      destinationID1:
        enabled: true
        user: "user"
        host: "host"
```

Reply:
@lvrach would this be OK or we would have issues with the SSH private key in this case as well?

Reply:
If we want to support this feature for general usage from multiple customers, it should be configurable for the control plane/app dashboard.

However, in this case, we want to experimentally (and quickly) support this for a single customer in an enterprise setup.